context/app/static/js/components/Search/ResultsTable.jsx

```diff
@@ -38,20 +39,23 @@ function makeTableBodyComponent(resultFields, detailsUrlPrefix, idField) {
       <>
         {hits.map((hit) => (
           <StyledTableBody key={hit._id}>
-            <StyledTableRow
-              className={'highlight' in hit && 'before-highlight'}
-              onClick={() => window.location.assign(detailsUrlPrefix + hit._source[idField])}
-              role="row link"
-            >
+            <StyledTableRow className={'highlight' in hit && 'before-highlight'}>
```

Comment:
```suggestion
            <StyledTableRow
              className={'highlight' in hit && 'before-highlight'}
              onClick={(event) => {
                if (
                  !window.getSelection().toString() && // If text selected, user doesn't want to navigate.
                  event.target.localName !== 'a' // Let <a> handle itself, for command-clicks.
                ) {
                  window.location.assign(detailsUrlPrefix + hit._source[idField]);
                }
              }}
              role="row link"
            >
```
Maybe?

Reply:
Per ui meeting, we're fine with just the ID link.